static/app/views/replays/detail/layout/splitPanel.tsx

```diff
@@ -0,0 +1,290 @@
+import {DOMAttributes, ReactNode, useCallback, useRef, useState} from 'react';
```

Comment:
Really crazy file that needs tests and love and all that. Until the bugs are found... I'm a proud baby-mama!

It'll probably take a few strategies to test it all, UI as well as breaking up some of the math stuff into utils.